### PR TITLE
task_52042_edit_profile

### PR DESCRIPTION
**Pull request by TMHuy01-08**
## Related Tickets
- [##52042](https://edu-redmine.sun-asterisk.vn/issues/52042)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- Xem và sửa thông tin cá nhân: Email / Name / Password

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by MinhHieu2804** [comment]
approved

**Comment by TMHuy01-08** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,6 +1,9 @@
 class UsersController < ApplicationController
+  before_action :find_user, except: %i(new create)
+  before_action :logged_in_user, only: %i(edit update)
+  before_action :correct_user, only: %i(edit update)
```
Comment (ngocvt-0484) [inline app/controllers/users_controller.rb]: gộp vs dòng trên

**Comment by TMHuy01-08** [comment]
ready

**Pull request merged**